deploy/ocp-templates/heketi-template.yaml

```diff
@@ -106,13 +99,16 @@ objects:
               port: 8080
         volumes:
         - name: db
-        - name: heketi-db-secret
-          secret:
-            secretName: heketi-db-backup
+          glusterfs:
+            endpoints: heketi-storage-endpoints
+            path: heketidbstorage
 parameters:
 - name: HEKETI_USER_KEY
   displayName: Heketi User Secret
   description: Set secret for those creating volumes as type _user_
 - name: HEKETI_ADMIN_KEY
   displayName: Heketi Administrator Secret
   description: Set secret for administration of the Heketi service as user _admin_
+- name: HEKETI_KUBE_CERTFILE
```

Comment:
In heketi 4 This environmental Variable is removed. So we do not need this section anymore.